test/k8sT/DatapathConfiguration.go

```diff
@@ -154,25 +154,28 @@ var _ = Describe("K8sDatapathConfig", func() {
 	})
 
 	Context("DirectRouting", func() {
-		directRoutingOptions := []string{
-			"--set global.tunnel=disabled",
-			"--set global.autoDirectNodeRoutes=true",
-		}
-
 		It("Check connectivity with automatic direct nodes routes", func() {
 			SkipIfFlannel()
 
-			deployCilium(directRoutingOptions)
+			deployCilium([]string{
+				"--set global.tunnel=disabled",
+				"--set global.autoDirectNodeRoutes=true",
+			})
+
 			Expect(testPodConnectivityAcrossNodes(kubectl)).Should(BeTrue(), "Connectivity test between nodes failed")
 			cleanService()
 		})
 
 		It("Check direct connectivity with per endpoint routes", func() {
 			SkipIfFlannel()
 
-			deployCilium(append(directRoutingOptions,
+			deployCilium([]string{
+				"--set global.tunnel=disabled",
+				"--set global.autoDirectNodeRoutes=true",
 				"--set global.endpointRoutes.enabled=true",
-			))
+				"--set global.ipv6.enabled=false",
```

Comment:
It seems that this was unintentionally added (IPv6 was not previously disabled in this test in v1.6)?

Reply:
Yes, true. I though I should stick to parameters you have in #10388, but I can remove it.